demosplan/DemosPlanCoreBundle/Controller/Forum/DemosPlanReleaseController.php

```diff
@@ -841,7 +845,7 @@ public function exportReleaseAction(Environment $twig, string $releaseId)
         $response->setContent($bom.$response->getContent());
         $filename = 'export_'.$part.'_'.date('Y_m_d_His').'.csv';
         $response->headers->set('Content-Type', 'text/csv');
-        $response->headers->set('Content-Disposition', $this->generateDownloadFilename($filename));
+        $response->headers->set('Content-Disposition', $pdfNameService->generateDownloadFilename($filename));
```

Comment:
Why did you choose PdfNameService? In this case it is used to generate a name for a csv file which seems weird

Reply:
I don't think so as in this case the service name would be weird when generating Filenames for pdf files. I think it needs to be something neutral

Reply:
done in https://github.com/demos-europe/demosplan-core/pull/1573/commits/ec3512bd9a27ab7cfbcb347b1555ce7ef448cced